### ISSUE TITLE
Share via Messages screen is missing a cancel button

### ISSUE THREAD

**Issue by farktronix** [RelistenNet/relisten-ios#161]
I tried to share the currently playing song via messages. There's no cancel button once the share screen appears.

![screen shot 2018-10-26 at 13 14 28](https://user-images.githubusercontent.com/4943/47590458-2dc78a00-d921-11e8-8e2b-03475bda99da.png)

**Comment by Maj0rPaine** [comment]
@farktronix is this related to Chameleon framework? I used it in another project and it completely hijacked all styling. I ultimately had to override the appearance proxy for affected classes.

**Comment by farktronix** [comment]
Good idea! I went ahead and removed the Chameleon framework to see if that was the problem, but this bug still happens. The changes are on a branch at https://github.com/farktronix/relisten-ios/tree/KarmaChameleon but I'm not sure if it's worth merging them into master.

**Comment by Maj0rPaine** [comment]
Yeah...I probably spent too much time trying to fix that bug. No matter what I tried it was impossible to set the appearance of MFMessageComposeController when it's not directly managed.

**Comment by alecgorge** [comment]
I haven't actually looked at this specific issue but in the past I think I've solved it by either:

1. setting a the tint color for toolbar buttons within `MFMessageComposeController`
2. only setting the custom tint colors for those within a UINavigationController subclass and then using that subclass everywhere in the main app
